### ISSUE TITLE
paddlex API训练可以用预训练模型吗

### ISSUE THREAD

**Issue by 15254042250** [PaddlePaddle/PaddleX#1435]
paddlex API训练可以用预训练模型吗

paddlex API可以导出预训练模型吗

num_classes = len(train_dataset.labels)
model = pdx.det.PPYOLOv2(num_classes=num_classes, backbone='ResNet50_vd_dcn')
model.train(
    num_epochs=170,
    train_dataset=train_dataset,
    train_batch_size=8,
    eval_dataset=eval_dataset,
    pretrain_weights='COCO',
    learning_rate=0.005 / 12,
    warmup_steps=1000,
    warmup_start_lr=0.0,
    lr_decay_epochs=[105, 135, 150],
    save_interval_epochs=5,
    save_dir='output/ppyolov2_r50vd_dcn')

**Comment by FlyingQianMM** [comment]
可以的，参考API文档说明，指定pretrain_weights即可。

该issue先关闭了哈，如有需要再打开～
